presto-native-execution/presto_cpp/main/http/HttpServer.cpp

```diff
@@ -76,6 +76,20 @@ void sendErrorResponse(
       .sendWithEOM();
 }
 
+HttpConfig::HttpConfig(const folly::SocketAddress& httpAddress)
+    : httpAddress_(httpAddress) {}
+
+proxygen::HTTPServer::IPConfig HttpConfig::httpIpConfig() const {
+  proxygen::HTTPServer::IPConfig ipConfig{
+      httpAddress_, proxygen::HTTPServer::Protocol::HTTP};
+  if (SystemConfig::instance()->httpServerReusePort()) {
```

Comment:
httpsConfig doesn't need this the port reuse handling? We could only have https config setup?